content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,25 +9,82 @@ weight: 20
+<!-- body -->
+## Kubernetes API terminology {#standard-api-terminology}
+
+Kubernetes generally leverages common RESTful terminology to describe the
+API concepts:
+
+* A *resource type* is the name used in the URL (`pods`, `namespaces`, `services`)
+* All resource types have a concrete representation (their object schema) which is called a *kind*
+* A list of instances of a resource is known as a *collection*
+* A single instance of a resource type is called a *resource*, and also usually represents an *object*
+* For some resource types, the API includes one or more *sub-resources*, which are represented as URI paths below the resource
+
+Most Kubernetes API resource types are
+[objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects):
+they represent a concrete instance of a concept on the cluster, like a
+pod or namespace. A smaller number of API resource types are *virtual* in
+that they often represent operations on objects, rather than objects, such
+as a permission check
+(use a POST with a JSON-encoded body of `SubjectAccessReview` to the
+`subjectaccessreviews` resource), or the `eviction` sub-resource of a Pod
+(used to trigger
+[API-initiated eviction](/docs/concepts/scheduling-eviction/api-eviction/)).
+
+### Object names
+
+All objects you can create via the API have a unique object
+{{< glossary_tooltip text="name" term_id="name" >}} to allow idempotent creation and
+retrieval, except that virtual resource types may not have unique names if they are
+not retrievable, or do not rely on idempotency.
+Within a {{< glossary_tooltip text="namespace" term_id="namespace" >}}, only one object
+of a given kind can have a given name at a time. However, if you delete the object,
+you can make a new object with the same name. Some objects are not namespaced (for
+example: Nodes), and so their names must be unique across the whole cluster.
+
+### API verbs
+
+Almost all object resource types support the standard HTTP verbs - GET, POST, PUT, PATCH,
+and DELETE. Kubernetes also uses its own verbs, which are often written lowercase to distinguish
+them from HTTP verbs.
+
+Kubernetes uses the term **list** to describe returning a [collection](#collections) of
+resources to distinguish from retrieving a single resource which is usually called
+a **get**. If you sent an HTTP GET request with the `?watch` query parameter,
+Kubernetes calls this a **watch** and not a **get** (see
+[Efficient detection of changes](#efficient-detection-of-changes) for more details).
+
+For PUT requests, Kubernetes internally classifies these as either **create** or **update**
+based on the state of the existing object. An **update** is different from a **patch**; the
+HTTP verb for a **patch** is PATCH.
+
+## Resource URIs
+
+All resource types are either scoped by the cluster (`/apis/GROUP/VERSION/*`) or to a
+namespace (`/apis/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource
```

Comment:
Could break up the sentence, "A namespace-scoped resource ..."
The API server (?) deletes the namespace-scoped resource when the namespace is deleted. Access to a resource type is controlled by authorization checks on the namespace scope.